content/jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest/presets/js-with-ts',
```

Comment:
I believe this can be only `ts-jest` now. Same goes for `functions` too. Since we don't import any js files at all anymore

Reply:
Fixed